web/src/components/core/Description.test.jsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { screen } from "@testing-library/react";
+import { plainRender } from "~/test-utils";
+import { Description } from "~/components/core";
+
+describe("Description", () => {
+  const description = "Some great description";
+  const item = "Item with description";
+
+  it("displays the description after clicking the object", async () => {
+    const { user } = plainRender(<Description description={description}>{item}</Description>);
+
+    // the description is not displayed just after the render
+    expect(screen.queryByText(description)).not.toBeInTheDocument();
+    expect(screen.queryByRole("dialog")).not.toBeInTheDocument();
+
+    // click it
+    const item_node = screen.getByText(item);
+    await user.click(item_node);
+
+    // then the description is visible in a dialog
+    screen.getByRole("dialog");
+    screen.getByText(description);
+  });
+
+  const empty_tester = async (subject) => {
```

Comment:
NP: until now we have been writing more explicit tests, not using this kind of helper functions because usually they saves keystrokes at the cost of doing them hard to read/follow. Anyway, I'm not against them. Actually, I have no strong opinion here. Just naming suggestions

* empty_tester -> expectNoDescription or something similar. Note the camelCase instead the snake_case
* subject -> content, which is what you are "plain rendering"
*

Reply:
The tests we like 95% the same code, repeating the same three times was ugly to me. So I created this shared function.